stubs/core.stub

```diff
@@ -67,3 +67,166 @@ function base64_encode(string $string) : string {}
  * @return ($string is non-empty-string ? non-empty-string : string)
  */
 function bin2hex(string $string): string {}
+
+/**
+ * @param array<mixed> $result
+ * @param-out array<int|string, string> $result
+ */
+function parse_str(string $string, array &$result): void {}
+
+/**
+ * @template T
+ * @template TArray as array<T>
+ *
+ * @param TArray $array
+ * @param-out (TArray is non-empty-array ? non-empty-list<T> : list<T>) $array
+ */
+function shuffle(array &$array): bool
+{
+}
+
+/**
+ * @template T
+ * @template TArray as array<T>
+ *
+ * @param TArray $array
+ * @param-out (TArray is non-empty-array ? non-empty-list<T> : list<T>) $array
+ */
+function sort(array &$array, int $flags = SORT_REGULAR): bool
+{
+}
+
+/**
+ * @template T
+ * @template TArray as array<T>
+ *
+ * @param TArray $array
+ * @param-out (TArray is non-empty-array ? non-empty-list<T> : list<T>) $array
+ */
+function rsort(array &$array, int $flags = SORT_REGULAR): bool
+{
+}
+
+/**
+ * @template T
+ * @template TArray as array<T>
+ *
+ * @param TArray $array
+ * @param callable(T,T):int $callback
+ * @param-out (TArray is non-empty-array ? non-empty-list<T> : list<T>) $array
+ */
+function usort(array &$array, callable $callback): bool
```

Comment:
There are already stubs for some of those functions, so you need to update an existing stub instead of adding a new one.

Reply:
done